styles.css

```diff
@@ -0,0 +1,108 @@
+* {
+  border: 0;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  color: #fff;
+  outline: none;
+  text-align: center;
+  background: none repeat scroll 0 0 transparent;
+  border-spacing: 0;
+  list-style-type: none;
+}
+
+.header {
+  margin: 1%;
+  padding: 1%;
+  width: 100%;
+  display: flex;
```

Comment:
[OPTIONAL] Consider removing the margin of 1% and also setting the padding to 1rem, that value makes it look more like the figma design.